go/database/mpt/state_test.go

```diff
@@ -403,7 +404,7 @@ func TestState_StateModificationsWithoutErrorHaveExpectedEffects(t *testing.T) {
 				t.Errorf("account should not exist: %v err: %s", exists, err)
 			}
 
-			var emptyBalance common.Balance
+			emptyBalance := amount.New()
```

Comment:
perhaps we should have also an alias `amont.Zero()`

Reply:
Once it's all merged, I'll create a new PR with `Zero` factory method and replaced all `New()` instances.